### ISSUE TITLE
Windows Desktop App won't remember window position

### ISSUE THREAD

**Issue by vanclute** [zulip/zulip-desktop#1066]
I run multiple displays, and keep all my social stuff on one particular screen, sized just the way I want them. But Zulip refuses to remember where I had it from one launch to the next. It always pops up on my primary display instead, and I have to reposition/scale it every time.

**Comment by matthewlswanson** [comment]
Yes! This frustrates me to no end. My other chat window automatically goes to the correct monitor and position on reboot but I have to drag Zulipchat over and resize it every time I reboot.

**Comment by matthewlswanson** [comment]
Any update on this? Was hoping it would be better in Windows 11 but unfortunately it is not
